src/storage-controller/src/lib.rs

```diff
@@ -2285,7 +2296,11 @@ where
                 Arc::new(UnitSchema),
             )
             .await;
-            let worker = persist_handles::PersistTableWriteWorker::new_txns(tx.clone(), txns);
+            let worker = persist_handles::PersistTableWriteWorker::new_txns(
+                tx.clone(),
+                txns,
+                lazy_persist_txn_tables,
+            );
             let txns_read = TxnsRead::start::<TxnsCodecRow>(txns_client.clone(), txns_id);
```

Comment:
Realized last night that having this txns_read around is wasteful for eager uppers (plus having it running increases our exposure to bugs slightly), so reworked a touch here to avoid that.